src/components/DocTypeDropdown/index.tsx

```diff
@@ -23,15 +28,15 @@ const createOptions = (opt: OptionItem) => (
 
 const renderOptions = (options: OptionItem[]) => options.map(createOptions);
 
-const Dropdown: React.FC<DropdownProps> = (props) => {
+const DocTypeDropdown: React.FC<DropdownProps> = (props) => {
 
-  const { className, name, options, title } = props;
-  const [dropdownValue, setDropdownValue] = React.useState<string | number>();
+  const { setDocType, className, name, options, title } = props;
+  const documentType = useSelector(state => state.createDoc[name]);
   const classes = dropdownStyles();
   const rootClass = classNames(classes.root, className);
 
   const handleChange = (event: React.ChangeEvent<{ value: any }>) => {
```

Comment:
Seeing this closely , this is actually a component ( I mean a reusable one ). So stuff like change handler should be provided as a prop, don't you think so ? . So this component can be reausable